shell/browser/native_window_mac.mm

```diff
@@ -1212,18 +1213,8 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
 }
 
 content::DesktopMediaID NativeWindowMac::GetDesktopMediaID() const {
-  auto desktop_media_id = content::DesktopMediaID(
-      content::DesktopMediaID::TYPE_WINDOW, GetAcceleratedWidget());
-  // c.f.
-  // https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/media/webrtc/native_desktop_media_list.cc;l=372?q=kWindowCaptureMacV2&ss=chromium
-  // Refs https://github.com/electron/electron/pull/30507
-  // TODO(deepak1556): Match upstream for `kWindowCaptureMacV2`
-#if 0
-    if (remote_cocoa::ScopedCGWindowID::Get(desktop_media_id.id)) {
-      desktop_media_id.window_id = desktop_media_id.id;
```

Comment:
Is this change not needed with the feature enabled ?

Reply:
removed!

Reply:
@deepak1556 we could do this, but then it would be invalid when for example passed into `getUserMedia.` I made this change for the same reason i did this one: https://github.com/electron/electron/pull/37014/files#diff-73f755a575b1c517f40cd8358386c43c4ffb0eff9e8a7a9ab80c5f50a248dcf4R156